### DISCUSSION TITLE
How to set a specific permission to read and write?

### DISCUSSION THREAD

**Discussion by IzumiSy** [community/community#26655] [Actions]
<p>I would like to set a specific permission for Github Action as read and write.</p>
<p>How can I do that? I tried some such as <code>read-write</code>, <code>read/write</code>, <code>read|write</code>, but all of them are said to be not valid syntax.</p>

**Comment by Simran-B** [comment, marked as answer]
<p>You can set permissions to <code>read</code>, <code>write</code>, or <code>none</code>. You can’t set multiple values, but that’s also unnecessary because <code>write</code> implies <code>read</code>.</p>

**Reply by dmitry-cello** [reply]
not very intuitive

**Reply by antoson** [reply]
> `write` implies `read`.

i am [astonished](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) :)

**Comment by feraudet** [comment]
How to set read-all + write if multiple values are unnecessary ?
